django/api/tests.py

```diff
@@ -77,3 +79,114 @@ def test_restaurants_default_order_by_name(self):
     def test_restaurant_filter_boundingbox(self):
         response = self.c.get("/api/restaurants/?in_bbox=-2,-2,2,2")
         self.assertEqual(response.data["count"], 2)
+
+
+class UserSerializerTestCase(TestCase):
+    def setUp(self):
+        self.test_user = UserFactory()
+        self.test_restaurant_1 = RestaurantFactory.create(
+            __sequence=1, owner=(self.test_user,)
+        )
+        self.test_restaurant_2 = RestaurantFactory.create(
+            __sequence=2, owner=(self.test_user,)
+        )
+
+    def test_user_owns_multiple_restaurants(self):
+        user = User.objects.get(username=self.test_user.username)
+        restaurants = Restaurant.objects.filter(owner=user)
+        self.assertEqual(2, len(restaurants))
+
+    def test_user_is_owner(self):
+        user = User.objects.get(username=self.test_user.username)
+        user_data = UserSerializer(user).data
+        restaurants = Restaurant.objects.filter(owner=user)
+        is_owner = True
+        for restaurant in restaurants:
+            owner = dict(
+                RestaurantSerializer(restaurant).data["properties"].pop("owner")[0]
+            )
+            if owner != user_data:
+                is_owner = False
+        self.assertEqual(True, is_owner)
+
+
+class UserViewsetTestCase(TestCase):
+    def setUp(self):
+        self.c = Client()
```

Comment:
```suggestion
        self.client = Client()
```

I see the motivation for wanting to keep this short because it gets used a lot, but I think the benefit of being clear about what `self.client` is for a reader coming along later is going to outweigh the benefit from faster typing. I think the best uses for one-character variable names are within small, tight loops, ideally where there is some convention or surrounding context immediately available to make the meaning clear. For example `[p.transformTo('4326') for p in polygon.points()]`.